pkgs/development/python-modules/nsenter/default.nix

```
@@ -0,0 +1,25 @@
+{ lib, buildPythonPackage, fetchPypi, contextlib2, argparse, flake8, pathlib, pythonOlder, pythonAtLeast }:
+
+buildPythonPackage rec {
+  pname = "nsenter";
+  version = "0.2";
+
+  propagatedBuildInputs = [
+    contextlib2
+    argparse
+    flake8
+  ] ++ lib.optional (pythonOlder "3.4") [
+    pathlib
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "1jr5592pshzn0ck75xb4f8dmcya8vb7x8byp9j7991fy0g5ihsl7";
+  };
```

generally src is listed before inputs
```suggestion

  src = fetchPypi {
    inherit pname version;
    sha256 = "1jr5592pshzn0ck75xb4f8dmcya8vb7x8byp9j7991fy0g5ihsl7";
  };
  
  propagatedBuildInputs = [
    contextlib2
    argparse
    flake8
  ] ++ lib.optional (pythonOlder "3.4") [
    pathlib
  ];

```

---

Thanks, implemented for all new packages.